staging/src/k8s.io/api/certificates/v1alpha1/types.go

```diff
@@ -23,6 +23,7 @@ import (
 // +genclient
 // +genclient:nonNamespaced
 // +k8s:prerelease-lifecycle-gen:introduced=1.26
+// +k8s:prerelease-lifecycle-gen:deprecated=1.34
```

Comment:
This will bump the removal to v1.37. If we anticipate removing it sooner, would recommend decreasing this number.

Reply:
can do that as a follow-up... alphas can be removed in a single release with no notice if needed by policy